### PR TITLE
set reasonable value for bonding_duration in chainspec staging config

### PR DESCRIPTION
**Pull request by mnaamani**
Fixes https://github.com/paritytech/substrate/issues/2288

Issue discovered by @bwhm

### CODE REVIEW

File: node/cli/src/chain_spec.rs [hunk 1]
```diff
@@ -112,7 +112,7 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 			current_session_reward: 0,
 			validator_count: 7,
 			sessions_per_era: 12,
-			bonding_duration: 60 * MINUTES,
+			bonding_duration: 1,
```
Comment (gavofyork) [inline node/cli/src/chain_spec.rs]: this isn't reasonable. bonding for staging network should be around 1 day.

File: node/cli/src/chain_spec.rs [hunk 2]
```diff
@@ -282,7 +282,7 @@ pub fn testnet_genesis(
 			minimum_validator_count: 1,
 			validator_count: 2,
 			sessions_per_era: 5,
-			bonding_duration: 2 * 60 * 12,
+			bonding_duration: 1,
```
Comment (gavofyork) [inline node/cli/src/chain_spec.rs]: same

File: node/cli/src/chain_spec.rs (hunk 1 above)
Comment (gavofyork) [reply node/cli/src/chain_spec.rs]:
```suggestion
			bonding_duration: 12,
```

File: srml/staking/src/lib.rs [hunk 3]
```diff
@@ -449,8 +449,8 @@ decl_storage! {
 		pub OfflineSlash get(offline_slash) config(): Perbill = Perbill::from_millionths(1000); // Perbill::from_fraction() is only for std, so use from_millionths().
 		/// Number of instances of offline reports before slashing begins for validators.
 		pub OfflineSlashGrace get(offline_slash_grace) config(): u32;
-		/// The length of the bonding duration in blocks.
-		pub BondingDuration get(bonding_duration) config(): T::BlockNumber = T::BlockNumber::sa(1000);
+		/// The length of the bonding duration in eras.
+		pub BondingDuration get(bonding_duration) config(): T::BlockNumber = T::BlockNumber::sa(1);
```
Comment (mnaamani) [inline srml/staking/src/lib.rs]: Should I perhaps set this default to 12 also ?

File: srml/staking/src/lib.rs [hunk 4]
```diff
@@ -469,8 +469,8 @@ decl_storage! {
...
```
Comment (gavofyork) [inline srml/staking/src/lib.rs]:
```suggestion
		pub BondingDuration get(bonding_duration) config(): T::BlockNumber = T::BlockNumber::sa(12);
```

**Comment by mnaamani** [comment]
CI job needs to be restarted gitlab-test-linux-stable

**Pull request merged**